README.md

```diff
@@ -88,3 +88,111 @@ git clone git@github.com:smashingboxes/tape.git
+* git and GitHub (or another remote host for your code)
+ * Here's a good article on how to get going with git: https://help.github.com/articles/set-up-git/.
+* ansible
+ * On a mac, we suggest using Homebrew to install ansible(`brew install ansible`). Otherwise, please refer to the ansible documentation: http://docs.ansible.com/ansible/intro_installation.html.
+
+### Rails Application Requirements
+
+Your rails application must:
+* use posgres as the database
+* use unicorn as the app server
+* have access to the taperole gem
+
+Usually, your Gemfile will include something like:
+```
+# Use postgresql as the database
+gem 'pg'
+
+# Use Unicorn as the app server
+gem 'unicorn'
+
+# Use taperole for deployment
+gem 'taperole', '~>1.3'
+
+```
+Note: You can also `$ gem install taperole` and not put Taperole in your Gemfile.
+
+### Your Host Server
+
+Below are instructions for creating a digital ocean droplet to host your application.
+This is not required, as long as you have root access via ssh to an Unbuntu precise64 (14.04 x64) server.
+
+1. Click "Create Droplet"
+2. Name your droplet. (For the purposes of this example, we'll be calling our droplet 'walkthrough'.)
+3. Select a size for your droplet. The smallest size is fine for our purposes, but if you're deploying a larger app, select whatever size is appropriate.
+4. Select a datacenter region.
+5. Choose Ubuntu 14.04.3 x64 as your image.
+6. Add your ssh key to the droplet.
+7. Create the droplet!
+
+To ensure you have ssh access as root to your server:
+`$ ssh root@0.0.0.0` should get you into the server.
+`$ pwd` in the server and you should see `/root`.
+`$ exit` to get close your session.
+
+### Deploying with Taperole
+
+1. Confirm that `taperole`, `pg`, and `unicorn` are in your gemfile.
+2. `$ bundle`
+3. When asked `Are you going to use vagrant? (y/n):`, say `n`.
+4. Make your hosts file. Assuming you're going to have multiple environments, it should look something like this:
+```
+[production]
+0.0.0.0 be_app_env=production be_app_branch=master
+
+[omnibox:children]
+production
+```
+5. Give yourself access to the server. This needs to be the same ssh key that you used on your digital ocean droplet. So create a file called <your_name>.pub in the dev_keys folder that was generated by tape installer install. `pbcopy < ~/.ssh/id_rsa.pub` in your terminal again, then paste the result into the file you just created.
+6. Update your tape_vars.yml file.
+```
+app_name: walkthrough
+
+be_app_repo: [git repo]
+
+dev_key_files:
+  - dev_keys/your_name.pub
+```
+7. Add this to your gitignore.
+```
+# Ignore this stuff for Taperole
+config/secrets.yml
+config/database.yml
+```
+8. If it isn't automatically, comment out front end stuff:
+ * `# fe_deploy` in omnibox.yml
+ * `# - frontend_deploy` in deploy.yml
+
+9. git add and commit and push to master
```

Comment:
Technically, this step isn't necessary, because tape will deploy using the config files in your current directory, but of course you'll want to share those up. Maybe add a note to that effect?